user-best-practices.md

```diff
@@ -0,0 +1,71 @@
+response to [forum post](https://forum.communityhealthtoolkit.org/t/best-practices-for-user-management-at-scale/1668)
+
+--------
+
+I've had a number of conversations about this over the past few weeks and I wanted to report back to the community about what I've found.  
+
+## How does the CHT recommend retiring users and replacing them with new ones?
+
+I found a number of real world practices that I wanted to spell out the pros and cons of. Let's start with
+
+### New contact & new user, associate with existing place
+
+This is top most recommended practice.  The steps are:
+
+1. Have the old CHW do one last sync to ensure all data is pushed up from their device
+1. Create a new contact for the new CHW.
+1. Make this contact the new primary contact for the old CHW's place - do not create a new place!
+1. Change the name the CHW's place as needed
+1. Create a new user  associated to the new contact.
+1. Change the password for the old CHW's user.
```

Comment:
The two steps about managing the old user (the last one, and the first one) are optional and sort of separate. You can have two CHWs in the same area with the same contacts. I would separate those two steps out under a subheading, for example...

```suggestion
1. Create a new contact for the new CHW.
1. Make this contact the new primary contact for the old CHW's place - do not create a new place!
1. Change the name the CHW's place as needed
1. Create a new user  associated to the new contact.

If you want to deactivate the old CHW

1. Have the old CHW do one last sync to ensure all data is pushed up from their device
1. Change the password for the old CHW's user on the server
1. Wipe the user's device to clear any private health information
```